plugins/pagerduty/api-report.md

```diff
@@ -40,43 +132,74 @@ export const PagerDutyCard: () => JSX.Element;
 //
 // @public (undocumented)
 export class PagerDutyClient implements PagerDutyApi {
-  // Warning: (ae-forgotten-export) The symbol "ClientApiConfig" needs to be exported by the entry point index.d.ts
   constructor(config: ClientApiConfig);
   // (undocumented)
   static fromConfig(
     configApi: ConfigApi,
-    discoveryApi: DiscoveryApi,
-    identityApi: IdentityApi,
+    { discoveryApi, fetchApi }: ClientApiDependencies,
   ): PagerDutyClient;
-  // Warning: (ae-forgotten-export) The symbol "ChangeEvent" needs to be exported by the entry point index.d.ts
-  //
   // (undocumented)
-  getChangeEventsByServiceId(serviceId: string): Promise<ChangeEvent[]>;
-  // Warning: (ae-forgotten-export) The symbol "Incident" needs to be exported by the entry point index.d.ts
-  //
+  getChangeEventsByServiceId(serviceId: string): Promise<ChangeEventsResponse>;
   // (undocumented)
-  getIncidentsByServiceId(serviceId: string): Promise<Incident[]>;
-  // Warning: (ae-forgotten-export) The symbol "OnCall" needs to be exported by the entry point index.d.ts
-  //
+  getIncidentsByServiceId(serviceId: string): Promise<IncidentsResponse>;
   // (undocumented)
-  getOnCallByPolicyId(policyId: string): Promise<OnCall[]>;
-  // Warning: (ae-forgotten-export) The symbol "Service" needs to be exported by the entry point index.d.ts
-  //
+  getOnCallByPolicyId(policyId: string): Promise<OnCallsResponse>;
   // (undocumented)
-  getServiceByIntegrationKey(integrationKey: string): Promise<Service[]>;
-  // Warning: (ae-forgotten-export) The symbol "TriggerAlarmRequest" needs to be exported by the entry point index.d.ts
-  //
+  getServiceByEntity(
+    pagerDutyEntity: PagerDutyEntity,
+  ): Promise<ServiceResponse>;
```

Comment:
Note quite what was intended here 😅, just pass the regular `Entity` straight from `useEntity` into this method, then do any unpacking of annotations and so on in the implementation.

Reply:
Btw, I realized there are other lookup and haven't investigated in-depth, but my assumption here is that we can use `getServiceByEntity` to look up the initial service, and then use the ID of that service in all the other places?